merged-packages/design-tokens/package.json

```diff
@@ -31,24 +31,17 @@
     "dist"
   ],
   "scripts": {
-    "build": "tsup --clean && yarn build:types && yarn build:css",
-    "build-storybook": "storybook build",
-    "build:clean": "rimraf dist && yarn build",
+    "build": "ts-bridge --project tsconfig.build.json --verbose --clean --no-references && yarn build:types && yarn build:css",
```

Comment:
Updating build script to align with other packages and adding build for types and css